playbook-website/app/javascript/site_styles/_site-style.scss

```diff
@@ -19,7 +19,6 @@ body {
   &--dark-mode-toggle-desktop {
     display: flex;
     justify-content: flex-end;
-    margin-top: $space_lg;
```

Comment:
I removed this in lieu of a bit of a structure change that I did, @nidaqg . See my Flex that wraps some things. Feel free to check if this looks odd in any spots.

Reply:
I like it @jasperfurniss , good enhancement!